src/app/account/additional-components/account-flows-tab/account-flows-subtabs/account-flows-datasets-subtab/account-flows-datasets-subtab.component.html

```diff
@@ -7,8 +7,45 @@
         <span class="fs-12 me-2 d-block"> Refresh now</span>
     </button>
 </div>
+
+<div class="my-4">
+    <mat-chip-listbox [(ngModel)]="flowsMode" (change)="onFlowsModeChange($event)">
+        <mat-chip-option [value]="ProcessCardGroup.ALL">All flows</mat-chip-option>
+        <mat-chip-option [value]="ProcessCardGroup.DATASETS">Datasets</mat-chip-option>
+        <mat-chip-option [value]="ProcessCardGroup.WEBHOOKS">Webhooks</mat-chip-option>
+    </mat-chip-listbox>
+</div>
+
+<div class="my-4 d-flex flex-row align-items-center" *ngIf="activeRollup">
+    <span class="fw-500 me-4">Rollup:</span>
+    <mat-button-toggle-group
+        [(ngModel)]="dashboardFiltersState.selectedFlowProcessStates"
+        (change)="onToggleRollup()"
+        multiple
+        class="me-4"
+        *ngIf="accountFlowsData.datasetsFiltersMode === ProcessCardFilterMode.CUSTOM; else defaultRollup"
+    >
+        <ng-container *ngFor="let item of ROLLUP_OPTIONS">
+            <mat-button-toggle [value]="item.state">
+                <mat-icon aria-hidden="true" class="fs-4 mr-2px mt-2px">{{ item.iconName }}</mat-icon
+                >{{ item.label }} {{ activeRollup[item.valueKey] }}</mat-button-toggle
+            >
+        </ng-container>
+    </mat-button-toggle-group>
+    <ng-template #defaultRollup>
+        <ng-container *ngFor="let item of ROLLUP_OPTIONS">
+            <div class="me-4 rollup-item border d-flex align-items-center">
+                <mat-icon aria-hidden="true" class="fs-4 mr-2px mt-2px">{{ item.iconName }}</mat-icon>
+                {{ item.label }} {{ activeRollup[item.valueKey] }}
+            </div>
+        </ng-container>
+    </ng-template>
+
+    <span class="fw-500 me-4">total: {{ activeRollup.total }}</span>
```

Comment:
Let's use Uppercase first letter words, for "Total" and the other options

Reply:
Fixed